src/assets.ts

```diff
@@ -428,4 +428,86 @@ export function addAssetsIfNecessary(server: OmnisharpServer): Promise<AddAssetR
         }).catch(err =>
             reject(err));
     });
+}
+
+function doAssetsExists(paths: Paths) {
```

Comment:
I might suggest we rename this method to 'doesAnyAssetExist' and/or add a comment that this returns true if we have a launch or tasks.json, since it isn't readily apparent from the name that this is doing a `||` rather than an `&&`.